React_Day5/avengers/src/Components/SearchBarComponent.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+
+import data from "../data";
+
+const searchAvengers = (searchText) => {
+  return data.filter((team)=> team.name.toLowerCase().includes(searchText));

```

Comment:
What  if team. name does not exist? It will break the application. Try using optional chaining.